src/js/plugins/search/view.js

```diff
@@ -265,7 +265,7 @@ class SearchView {
     toolbarSearch.classList.add('BRtoolbarSection', 'BRtoolbarSectionSearch');
     toolbarSearch.innerHTML = `
       <form class="BRbooksearch desktop">
-        <input type="search" name="query" class="BRsearchInput" val="" placeholder="Search inside"/>
+        <input type="search" name="query" class="BRsearchInput" val="" placeholder="Search inside"></input>
```

Comment:
QUESTION: Why the closing `</input>` tag? I believe the HTML5 spec says it shouldn't be there.

Reply:
that's what I said to myself 😅 
the chrome dev console was barking about the need for a closing tag at this line, removed.